keys/constants.go

```diff
@@ -177,7 +173,9 @@ var (
 	StatusStorePrefix = MakeKey(StatusPrefix, proto.Key("store-"))
 	// StatusNodePrefix stores all status info for nodes.
 	StatusNodePrefix = MakeKey(StatusPrefix, proto.Key("node-"))
-	// TableDataPrefix prefixes all Table data to aid in transitioning
-	// key:value data to Table data, and for ease of debugging.
-	TableDataPrefix = proto.Key("table-")
+
+	// TableDataPrefix prefixes all table data. It is specifically chosen to
+	// occur after the range of common user data prefixes so that tests which use
+	// those prefixes will not see table data.
+	TableDataPrefix = proto.Key("\xff")
```

Comment:
kind of weird to wedge this in before `EndKey`, are there plans to move this back to `tbl-` or something like that?